### ISSUE TITLE
Unable to get lynx working with the default example - Dynamo failed to run FX node with fake tensors: call_function <built-in method cat of type object at 0x2b00f79cc4c0>

### ISSUE THREAD

**Issue by TheMindExpansionNetwork** [kijai/ComfyUI-WanVideoWrapper#1403]
Hello been trying to wrap my head around this

I am using modal trying to get this deployed on my script

I get this error

WanVideoSampler
Dynamo failed to run FX node with fake tensors: call_function <built-in method cat of type object at 0x2b00f79cc4c0>(*([FakeTensor(..., device='cuda:0', size=(1, 16, 2048), dtype=torch.float16), Parameter(FakeTensor(..., device='cuda:0', size=(1, 16, 5120), dtype=torch.float16))],), **{'dim': 1}): got RuntimeError('Sizes of tensors must match except in dimension 1. Expected 2048 in dimension 2 but got 5120 for tensor number 1 in the list')

from user code:
File "/root/comfy/ComfyUI/custom_nodes/ComfyUI-WanVideoWrapper/wanvideo/modules/model.py", line 656, in torch_dynamo_resume_in_forward_at_646
lynx_x_ip = self.ip_adapter(self, q, lynx_x_ip)
File "/root/comfy/ComfyUI/custom_nodes/ComfyUI-WanVideoWrapper/lynx/modules.py", line 37, in forward
ip_x = torch.cat([ip_x, self.registers], dim=1)

Set TORCHDYNAMO_VERBOSE=1 for the internal stack trace (please do this especially if you're reporting a bug to PyTorch). For even more developer context, set TORCH_LOGS="+dynamo"

Not sure what the issue could be let me know you need anymore information

**Comment by kijai** [comment]
> RuntimeError('Sizes of tensors must match except in dimension 1. Expected 2048 in dimension 2 but got 5120 for tensor number 1 in the list')

Seems like model mismatch, are you using the full IP adapter model or the lite? If lite the resampler must also be the lite version, and with full it needs to be the full resampler.
